chapters/001-getting-started/README.md

```diff
@@ -180,7 +197,7 @@ You'll want to repeat this process on the other container (but in this case the
 
 ### Test the network connection
 
-On the CONTAINER_1 machine, run `tcpdump -n -i eth1`. This will run a program which "sniffs" ethernet frames on the same network interface that we just added the 10.1.1.3 address to. The `-n` flag to `tcpdump` tells that program not to try to resolve hostnames via DNS. The `-i eth1` flag tells `tcpdump` which interface to use.
+On the `pippin` machine, run `tcpdump -n -i eth1`. This will run a program which "sniffs" ethernet frames on the same network interface that we just added the 10.1.1.3 address to. The `-n` flag to `tcpdump` tells that program not to try to resolve hostnames via DNS. The `-i eth1` flag tells `tcpdump` which interface to use.
```

Comment:
> The `-i eth1` flag tells `tcpdump` which interface to use.

Perhaps which **network** interface to use.

(also: not necessary since this machine has only one interface?)